t/0700-unifyfs-stage-full.t

```diff
@@ -14,51 +14,64 @@ test_expect_success "testing temp dir exists" '
     test_path_is_dir  ${UNIFYFS_TEST_TMPDIR}
 '
 
-mkdir -p ${UNIFYFS_TEST_TMPDIR}/config_0700
-mkdir -p ${UNIFYFS_TEST_TMPDIR}/stage_source
-mkdir -p ${UNIFYFS_TEST_TMPDIR}/stage_destination_0700
+stage_cfg_dir=${UNIFYFS_TEST_TMPDIR}/stage/config_0700
+stage_src_dir=${UNIFYFS_TEST_TMPDIR}/stage/source
+stage_dst_dir=${UNIFYFS_TEST_TMPDIR}/stage/destination_0700
+mkdir -p $stage_cfg_dir $stage_src_dir $stage_dst_dir
 
 test_expect_success "stage testing dirs exist" '
-    test_path_is_dir  ${UNIFYFS_TEST_TMPDIR}/config_0700
-    test_path_is_dir  ${UNIFYFS_TEST_TMPDIR}/stage_source
-    test_path_is_dir  ${UNIFYFS_TEST_TMPDIR}/stage_destination_0700
+    test_path_is_dir  $stage_cfg_dir
+    test_path_is_dir  $stage_src_dir
+    test_path_is_dir  $stage_dst_dir
```

Comment:
It dawned on me that this bug might be in these tests as well and I was about to test it when I saw you were already adjusting these.

When doing multiple tests inside `test_expect_success`, we should chain them together with `&&`. `test_expect_success` determines pass/fail from the return code of the last command, so if the first command here were to fail, the return code would be lost on whatever the result of the second command is.


```suggestion
    test_path_is_dir  $stage_cfg_dir &&
    test_path_is_dir  $stage_src_dir &&
    test_path_is_dir  $stage_dst_dir
```
Probably won't help with why the last test with`test_might_fail` is failing, unless this shows something else is weird in Travis.

Reply:
I'm making these changes in the branch that I'm trying to get synchronized between catalyst and summit.  But if we think this might have bearing on the errors that Travis is throwing about the 0700 script, but all means, make the changes here.  I can merge them in easily enough.